.github/workflows/udmp-parser.yml

```diff
@@ -2,74 +2,117 @@ name: Builds
 
 on: [push, pull_request]
 
+
 jobs:
   Windows:
-    runs-on: windows-latest 
+    runs-on: windows-latest
     strategy:
       fail-fast: false
       matrix:
         generator: ['ninja', 'msvc']
-        arch: ['win32', 'win64']
+        arch: ['x64'] #, 'x86', 'arm64']
```

Comment:
I have never had issues with other archs but actions does a cartesian product of all your matrix settings. That's why I only enabled x64, but I will enable the other ones. Maybe we can diminish the load by dropping ninja build?

Reply:
I think it would be useful to build other archs as it's easy to break them - but also if it's too much of a pain to get working don't worry about it :) I can take that pain and figure it out later